packages/html/src/pager/tests/pager.tsx

```diff
@@ -7,25 +8,25 @@ export default () =>(
 
             <span>LTR</span>
             <section>
-                <Pager />
+                <PagerNumeric />
             </section>
             <section>
-                <Pager type="input" focus />
+                <PagerInput focus />
```

Comment:
We could use states from the **pager.spec** - https://github.com/telerik/kendo-themes/blob/develop/packages/html/src/pager/pager.spec.tsx#L9

Like here: https://github.com/telerik/kendo-themes/blob/develop/packages/html/src/combobox/tests/combobox-solid.tsx#L25